### PR TITLE
Dockcross 20240418 88c04a4

### PR DESCRIPTION
**Pull request by thewtex**

### CODE REVIEW

File: packages/dicom/dcmtk/apply-presentation-state-to-image.cxx [hunk 1]
```diff
@@ -167,7 +169,10 @@ static void dumpPresentationState(STD_NAMESPACE ostream &out, DVPresentationStat
   }
 
   // ICC color Profile
-  const OFVector<Uint8> iccProfile = ps.getICCProfile();
+  IODImagePixelModule<Uint16> iodImagePixelModule;
+  iodImagePixelModule.read(dset);
+  OFVector<Uint8> iccProfile;
```
Comment (thewtex) [inline packages/dicom/dcmtk/apply-presentation-state-to-image.cxx:174]: findAndGetUint8Array(DCM_ICCProfile, values)

Comment (thewtex) [reply packages/dicom/dcmtk/apply-presentation-state-to-image.cxx:174]: const Uin8* data;
dset->findAndGetUint8Arra(DCM_ICCProfile, data)

Comment (michaelonken) [reply packages/dicom/dcmtk/apply-presentation-state-to-image.cxx:174]: @thewtex Please look at [my fork of this](https://github.com/InsightSoftwareConsortium/itk-wasm/compare/main...michaelonken:itk-wasm:dockcross-20240418-88c04a4) where I fixed access to the ICC profile. Please also look at the comment on the last (=my) commit in that branch.

Comment (thewtex) [reply packages/dicom/dcmtk/apply-presentation-state-to-image.cxx:174]: @michaelonken awesome! Thank you! I followed up on your commit's comment.

**Pull request merged**